package/MDAnalysis/core/groups.py

```diff
@@ -2586,27 +2639,30 @@ def write(self, filename=None, file_format="PDB",
             format = format or file_format
             format = format.strip().upper()
 
-            multiframe = kwargs.pop('multiframe', None)
-
             writer = get_writer_for(filename, format=format, multiframe=multiframe)
-            #MDAnalysis.coordinates.writer(filename, **kwargs)
-            coords = True
         except (ValueError, TypeError):
-            coords = False
+            pass
+        else:
+            with writer(filename, n_atoms=self.n_atoms, **kwargs) as w:
```

Comment:
Can you do this in try/except/finally so that you always restore the trajectory to the current frame at which we were before we wrote the trajectory?

This whole thing is definitely not thread safe (unless you were to make a copy of the Universe before writing) but that's not a major concern at the moment.